include/layers/InputLayer.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <stdexcept>
+#include <string>
+
+#include "layers/Layer.hpp"
+
+class InputLayer : public Layer {
+ public:
+  InputLayer() = default;
+  static void run(const std::vector<std::string>& path, Tensor& output) {
```

Comment:
run() should accept input tensor only. Image should be read separately outside any layer